tidb-cloud/tidb-cloud-billing-dm.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Data Migration Billing
+summary: Learn about billing for Data Migration in TiDB Cloud.
+---
+
+# Data Migration Billing
+
+This document describes the billing for Data Migration in TiDB Cloud.
+
+## Specifications for Data Migration
+
+TiDB Cloud measures the capacity of Data Migration in Replication Capacity Units (RCUs). When you create a data migration job, you can select an appropriate specification. You will be charged for these Data Migration RCUs.
+
+TiDB Cloud measures incremental data migration in rows/s. This unit indicates the number of rows per second generated by incremental replication Data Manipulation Language (DML) statements. It also means if the upstream executes INSERT, UPDATE, or DELETE statements within 10000 rows, the Data Migration job can replicate the 10000 rows to the downstream within 1 second.
+
+All the following performance values are maximum performances. Note that the performance values are for reference only and might vary in different scenarios.
+
+The following table lists the specifications for Data Migration:
+
+| Specification (RCUs)| Full data migration | Incremental data migration (rows/s)|
+|---------------|---------------------|----------------------------|
+| 2   | 25 MiB/s | 10000  |
+| 4   | 35 MiB/s | 20000 |
+| 8   | 40 MiB/s | 40000 |
+| 16  | 45 MiB/s | 80000 |
+
+## Price
+
+The following table lists the price of TiDB Cloud for each Data Migration RCU:
+
+| Cloud provider | Region                      | RCU price ($/hr) |
+|----------------|-----------------------------|------------------|
+| AWS            | Oregon (us-west-2)          |          $0.1307 |
+| AWS            | Singapore (ap-southeast-1)  |          $0.1623 |
+| AWS            | Tokyo (ap-northeast-1)      |          $0.1669 |
+
```

Comment:
other regions on AWS also will be released at 27,DEC

Reply:
OK  added all.